test/models/thesis_test.rb

```diff
@@ -1004,4 +1004,19 @@ def attach_file_with_purpose_to(thesis, purpose = 'thesis_pdf')
     assert_equal 1, thesis.dspace_metadata.attachments.length
     assert thesis.valid?
   end
+
+  test 'evaluates whether a user is a student' do
```

Comment:
Is there a reason this is part of the thesis test suite, and not the user test suite?

Reply:
Ha! Nope. I'll fix that.